scripts/leaf-scripts/filehistorgram.php

```diff
@@ -0,0 +1,54 @@
+<?php
+error_reporting(E_ALL);
+ini_set('display_errors', 1);
+require_once 'globals.php';
+require_once LIB_PATH . '/php-commons/Db.php';

```

Comment:
Does this pathing still work with current master @jampaul3 ?

Reply:
I would do 
require_once getenv('APP_LIBS_PATH') . '/globals.php';
require_once getenv('APP_LIBS_PATH') . '../Leaf/Db.php';